config/config.example.yml

```diff
@@ -1,13 +1,757 @@
+## only for a single function.
+##
+## Accepted values: true, false, dash, livestreams, downloads, local
+## Default: false
+##
+#disable_proxy: false
+
+##
+## Size of the HTTP pool used to connect to youtube. Each
+## domain ('youtube.com', 'ytimg.com', ...) has its own pool.
+##
+## Accepted values: a positive integer
+## Default: 100
+##
+#pool_size: 100
+
+##
+## Enable/Disable the use of QUIC (HTTP/3) when connecting
+## to the youtube API and websites ('youtube.com', 'ytimg.com').
+##
+## Accepted values: true, false
+## Default: true
+##
+#use_quic: true
+
+##
+## Additionnal cookies to be sent when requesting the youtube API.
+##
+## Accepted values: a string in the format "name1=value1; name2=value2..."
+## Default: <none>
+##
+#cookies:
+
+##
+## Force connection to youtube over a specific IP family.
+##
+## Note: This may sometimes resolve issues involving rate-limiting.
+## See https://github.com/ytdl-org/youtube-dl/issues/21729.
+##
+## Accepted values: ipv4, ipv6
+## Default: <none>
+##
+#force_resolve:
+
+
+# -----------------------------
+#  Logging
+# -----------------------------
+
+##
+## Path to log file. Can be absolute or relative to the invidious
+## binary. This is overriden if "-o OUTPUT" or "--output=OUTPUT"
+## are passed on the command line.
+##
+## Accepted values: a filesystem path or 'STDOUT'
+## Default: STDOUT
+##
+#output: STDOUT
+
+##
+## Logging Verbosity. This is overriden if "-l LEVEL" or
+## "--log-level=LEVEL" are passed on the command line.
+##
+## Accepted values:
+##   LogLevel::All    (Log everything)
+##   LogLevel::Trace
+##   LogLevel::Debug
+##   LogLevel::Info
+##   LogLevel::Warn
+##   LogLevel::Error
+##   LogLevel::Fatal
+##   LogLevel::Off    (Disable log output)
+## Default: LogLevel::Info
+##
+#log_level: LogLevel::Info
+
+
+# -----------------------------
+#  Features
+# -----------------------------
+
+##
+## Enable/Disable the "Popular" tab on the main page.
+##
+## Accepted values: true, false
+## Default: true
+##
+#popular_enabled: true
+
+##
+## Enable/Disable statstics (available at /api/v1/stats).
+## The following data is available:
+##   - Software name ("invidious") and version+branch (same data as
+##     displayed in the footer, e.g: "2021.05.13-75e5b49" / "master")
+##   - The value of the 'registration_enabled' config (true/false)
+##   - Number of currently connected users
+##   - Average Number of connected users (last month & last 6 months)
+##   - Timestamp of the last server restart
+##   - Timestamp of the last "Channel Refresh" job execution
+##
+## Warning: This setting MUST be set to true if you plan to run
+## a public instance. It is used by api.invidious.io to refresh
+## your instance's status.
+##
+## Accepted values: true, false
+## Default: false
+##
+#statistics_enabled: false
+
+
+# -----------------------------
+#  Users and accounts
+# -----------------------------
+
+##
+## Allow/Forbid Invidious (local) account creation. Invidious
```

Comment:
Why "local"? It forbids all accounts creation.

\+ 

Accounts, not account

Reply:
local, as in accounts that belong to this instance (AFAIK, those aren't shared across instances).

Reply:
and "account" is singular because it's a general form.

Reply:
>local, as in accounts that belong to this instance (AFAIK, those aren't shared across instances).

It assume that it doesn't forbid the google accounts feature

>and "account" is singular because it's a general form.

I feel like "accounts" is more correct, however I'm not a native english speaker, so I'm maybe wrong.

Reply:
> It assume that it doesn't forbid the google accounts feature

It doesn't, yeah. Registration is only for invidious accounts. You can still connect to Google.

> I feel like "accounts" is more correct, however I'm not a native english speaker, so I'm maybe wrong.

It's the same in french: "Activer/Désactiver la création de compte utilisateur" <- singular

Reply:
>It doesn't, yeah. Registration is only for invidious accounts. You can still connect to Google.

Oh... this isn't a good thing.

>It's the same in french: "Activer/Désactiver la création de compte utilisateur" <- singular

I guess...

Reply:
> Oh... this isn't a good thing.

Why? `login_enabled: false` forbids all types of login (Google/Invidious), and `enable_registration: false` disables all types of registration (i.e Invidious, because it's the only one available).

Reply:
Disabling registration should disable all registration of new users, the problem is that now, if I understand what your saying, people can still register with a Google account. Meaning that the only way to fully disable registration, is to disable login... meaning existing user aren't able to login. It's a problem.

Reply:
> Disabling registration should disable all registration of new users, the problem is that now, if I understand what your saying, people can still register with a Google account. Meaning that the only way to fully disable registration, is to disable login... meaning existing user aren't able to login. It's a problem.

Ah, I should have been more explicit about it: you can't register a Google account from invidious itself. You have to do so on Google's website. So we have no impact on it from config ^^

Reply:
>you can't register a Google account from invidious itself. You have to do so on Google's website. So we have no impact on it from config

Of course, but the data of an user connecting via google is still stored on the server. 

Unless the google login just "brings" the user information to invidious and an actual registration is required?

Reply:
> Of course, but the data of an user connecting via google is still stored on the server.

Ah, yes.